src/math/constants.js

```diff
@@ -11,6 +13,7 @@ const spatialResolution = 1e5
 const EPS = 1e-5
 
 module.exports = {
+  TWOPI,
```

Comment:
could you please elaborate on this one ?

Reply:
I got tired of writing Math.Pi*2 (radians) which is the replacement for 360 (degrees)
I can revert if you don’t like it.

Reply:
Reverted.